### ISSUE TITLE
Finetuning Protocols on ConvNeXt-Large Backbone

### ISSUE THREAD

**Issue by qsisi** [YvanYin/Metric3D#195]
Hello! Thanks for open-sourcing this amazing project.

I'm currently finetuning the ConvNeXt-L models on a custom dataset, could you provide some advice on the finetuning protocols? Such as learning rate, iters, loss choices, etc. 

Thanks for your help!

**Comment by YvanYin** [comment]
You can use lr=1e-4 or even smaller lr for fine-tuning. Iters depends on your data size. Losses can follow our papers argued ones.
